res/css/photon-components.css

```diff
@@ -121,3 +154,139 @@
 .photon-message-bar-error .photon-button:hover:active {
   background-color: var(--red-90);
 }
+
+/* Label */
+
+.photon-label,
+.photon-label-micro {
+  display: inline-flex;
+  color: var(--grey-90);
+}
+
+.photon-label {
+  padding: 3px 0;
+  font-size: 13px;
+}
+
+.photon-label-micro {
+  padding: 3px 0;
```

Comment:
The padding is the same in both flavors, maybe it should go in the previous rule ?